src/shared/components/msktable/MSKTable.tsx

```diff
@@ -436,27 +437,37 @@ export default class MSKTable<T> extends React.Component<MSKTableProps<T>, {}> {
         />
     }
 
-    render() {
+    buildTextBetweenTheButtons(): string {
```

Comment:
I think we should make `@computed get textBetweenButtons` and put it in `MSKTableStore`